apps/src/code-studio/components/header/HeaderPopup.jsx

```diff
@@ -63,54 +63,57 @@ export default class HeaderPopup extends Component {
       },
       {includeUserId: true}
     );
+
+    $(document).on('click', this.handleClickDocument);
```

Comment:
setting these event handlers to "capture" might fix some of the issues brad listed, so that this handler has a chance to run before another event handler swallows the event.

Reply:
That's a good idea.  I wouldn't getting this PR in first to get us back to parity, and then trying that in a followup, since I feel like using "capture" at the document-level would need some pretty broad testing to make sure there are no inadvertent side effects.